autoagora/utils/constants.py

```diff
@@ -14,6 +14,13 @@
 # stddev time:  {{frequent_query.stddev_time}}
 {{frequent_query.query}} => {{frequent_query.avg_time}} * $GLOBAL_COST_MULTIPLIER;
 {% endfor %}
+{% if manual_entry is defined %}
 
-default => $DEFAULT_COST * $GLOBAL_COST_MULTIPLIER;\
+{{manual_entry}}
+
+default => $DEFAULT_COST * $GLOBAL_COST_MULTIPLIER;
+{% else %}
+default => $DEFAULT_COST * $GLOBAL_COST_MULTIPLIER;
```

Comment:
That default entry should always be there

Reply:
Right I see what you mean, did it in this way for the formatting but yeah I can just put it afterwards